app/views/spotlight/compounds/_document_admin_table.html.erb

```diff
@@ -0,0 +1,34 @@
+<%# container for all documents in index view -%>
+<table id="documents" class="table" data-turbolinks="false">
+  <thead>
+    <tr>
+      <th class="col-sm-1"></th>
+      <th class="col-sm-5"><%= t(:'spotlight.catalog.fields.title') %></th>
+      <th class="col-sm-2"><%= t(:'spotlight.catalog.fields.date_added') %></th>
+      <th class="col-sm-1 checkbox-toggle"><%= t(:'spotlight.catalog.fields.visibility') %></th>
+    </tr>
+  </thead>
+  <% @documents.each do |document| %>
+  <tr>	
+  <td><a href="<%=exhibit_solr_document_path({exhibit_id: @exhibit.slug, id: document._source[:id]})%>">
```

Comment:
In general, we prefer using methods on `SolrDocument`, instead of digging into `_source`. For example, `document._source[:id]` is better accessed as `document.id` or `document._source[]` as `document[]`